libraries/service_helpers.rb

```diff
@@ -0,0 +1,58 @@
+module CernerSplunk
+  # Mixin Helper methods for the Splunk Service resource
+  module ServiceHelpers
+    include ResourceHelpers
+    require 'win32/service' if RUBY_PLATFORM =~ /mswin|mingw|windows/
+
+    # Reports if the Windows service for the current package exists
+    # @return [Boolean]
+    def service_exists
```

Comment:
I can change to service_exists? but not service_running? and I can't figure out why:
```
Failure/Error:
       ChefSpec::SoloRunner.new({ step_into: ['splunk_service'] }.merge!(runner_params)) do |node|
         node.normal['test_parameters'] = test_params
         node.normal['run_state'].merge!(mock_run_state)
       end.converge('cerner_splunk_ingredient_test::service_unit_test')
     
     SyntaxError:
       splunk_service[splunk] (cerner_splunk_ingredient_test::service_unit_test line 5) had an error: SyntaxError: /Users/as032696/.rvm/rubies/ruby-2.1.8/lib/ruby/2.1.0/forwardable.rb:181: syntax error, unexpected '(', expecting ';' or '\n'
             def __service_running?_without_any_instance__(*args, &block)
                                                           ^
       /Users/as032696/.rvm/rubies/ruby-2.1.8/lib/ruby/2.1.0/forwardable.rb:181: syntax error, unexpected &
             def __service_running?_without_any_instance__(*args, &block)
                                                                   ^
```